capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/provider/UserIdProvider.kt

```diff
@@ -0,0 +1,23 @@
+package net.gini.android.capture.internal.provider
+
+import android.content.Context
+import java.util.UUID
+
+internal class UserIdProvider(context: Context) {
+
+    companion object {
+        private const val SP_NAME = "UserIdPref"
+        private const val KEY_USER_ID = "user_id"
+    }
+
+    private val sp = context.getSharedPreferences(SP_NAME, Context.MODE_PRIVATE)
+
+    fun getUserId(): String = sp.getString(
+        KEY_USER_ID, null
+    ) ?: UUID.randomUUID().toString().also { setUserId(it) }
```

Comment:
[Comment] Make sure that `net.gini.android.capture.internal.provider.InstallationIdProvider` don't do the same thing, please